.rubocop.yml

```diff
@@ -89,22 +93,11 @@ Style/RaiseArgs:
     - lib/rspec/core/hooks.rb
     - lib/rspec/core/option_parser.rb
     - lib/rspec/core/pending.rb
-
-Performance/RedundantBlockCall:
-  Exclude:
-    - lib/rspec/core/rake_task.rb
+    - spec/rspec/core/formatters/base_text_formatter_spec.rb
 
 Lint/IneffectiveAccessModifier:
   Exclude:
     - lib/rspec/core/memoized_helpers.rb # Fixing this file was too much of a diff
 
-Performance/Caller:
-  Exclude:
-    # The suggested fix for this cop only works on more recent rubies. The
-    # speed up can be substantial (previous optimization work I've done on
-    # RSpec involved caller optimization) but I haven't benchmarked this
-    # particular case to see if it's worth introducing version branching.
-    - 'lib/rspec/core/example_group.rb'
-
 Metrics/BlockLength:
-  Max: 193
+  Max: 2373
```

Comment:
Is it coming from `spec/**/*_spec.rb`?

Reply:
Increased to 206 because of:
```
lib/rspec/core/option_parser.rb:40:7: C: Metrics/BlockLength: Block has too many lines. [206/100]
      OptionParser.new do |parser| ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```